experiments/aldp/train.py

```diff
@@ -199,9 +199,9 @@
     rb_config = config['training']['replay_buffer']
     if rb_config['type'] == 'uniform':
         def initial_sampler():
-            x, log_w = model.annealed_importance_sampler.sample_and_log_weights(
+            point, log_w = model.annealed_importance_sampler.sample_and_log_weights(
                 batch_size, logging=False)
-            return x, log_w
+            return point.x, log_w
```

Comment:
AIS now returns an instance `Point` see `fab/sampling_methods/base.py` where this is now defined